works.yaml

```diff
@@ -1,4 +1,68 @@
 ---
+Darknet:
+  uri:
+  - http://pjreddie.com/darknet
+  description:
+  - Darknet is an open source neural network framework written in C and CUDA. It is fast, easy to install, and supports CPU and GPU computation.
+  license:
+  - MIT/X11
+  - WTFPL
+  license_reference:
+  - https://github.com/pjreddie/darknet/blob/master/LICENSE.mit
+  - https://github.com/pjreddie/darknet/blob/master/LICENSE.fuck
+  tags:
+  - go (game)
+  - neural network
+  - c
+  - multilicensed
+  - software
+Ray:
+  uri:
+  - https://github.com/zakki/Ray
+  description:
+  - Computer Go Program
+  license:
+  - Simplified BSD License
+  license_reference:
+  - https://github.com/zakki/Ray/blob/nn/COPYING
+  tags:
+  - game
+  - go (game)
+  - neural network
+  - c++
+  - software
+AQ:
+  uri:
+  - https://github.com/ymgaq/AQ
+  description:
+  - AQ is a program of Go game with level of expert players.
```

Comment:
I would appreciate it if you edited this to make it a more coherent sentence.  The original description in the project readme is kinda difficult to understand.  You might consider submitting a fix for the sentence structure/grammar to the project, too.